swagger.yaml

```diff
@@ -751,7 +751,99 @@ paths:
                 items:
                   $ref: "#/components/schemas/Error"
 
-  "/vacationsRequests":
+  "/severa/projects/{severaProjectGuid}/phaseswithhierarchy":
+    get:
+      operationId: getPhasesBySeveraProjectId
+      summary: Get all the phases for project
+      description: Retrieve all the phases for a particular project with given ID.
+      tags:
+        - Phases
+      parameters:
+        - name: severaProjectGuid
+          in: path
+          required: true
+          description: ID of the project.
+          schema:
+            type: string
+      responses:
+        "200":
+          description: Phases entry retrieved successfully
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Phase"
+        "default":
+          description: Invalid request was sent to the server.
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/Error"
+
+  "/severa/{severaUserGuid}/resourceallocations":
```

Comment:
Use camelCase for the endpoint naming e.g. resourceAllocations

Reply:
Does it matter how they are written in severa? These endpoint were "copied" from the docs